src/emissions.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import numpy as np
+# import geojson
+import os
+
+# os.chdir('C:/Data/ACT_GIS/STEEP/') # for Eli
+os.chdir('/home/alena/my_project/STEEP/')  # for Alena
+
+##required inputs: Car_parameters (as dictionary), segment speed(as a value)
+##outputs of emissionfactor fuction will be a g/km value which can be used to calculate the emissions of a segment
+# emissions = pd.read_csv('data/EC_Factors_Passenger.csv')
+# with open("static\TestRoute_Wag_Roest.geojson") as gj_file:
+#  gj = geojson.load(gj_file)
+# speeds = []
+# for feature in gj['features']:
+#   print(feature['attributes'])
+
+# route =
+# speed = sample.geojson[]
+# print(emissions.head())
+
+# final formula (Alpha * Speed ^ 2 + Beta * Speed + Gamma + (Delta/Speed) ) / (Epsilon * Speed^2 + Zita * Speed + Hta)
+
+# input will be speeds for each segment in csv
+# path is a path to the file
+# def constructSpeedDF (path):
+# spDF = read_csv(str(path))
+
+# create a sample dataFrame with speeds per segment
+
+speedPerSegmentDict = {'segment': [0, 1, 2, 3, 4],
+                       'Speed': [9, 70, 55, 80, 70]}
+
+speedPerSegmentDF = pd.DataFrame(speedPerSegmentDict)
+speedPerSegmentDF.to_csv('data/b_SpeedFlow_sample.csv')
+
+# this function derive values for further emission factors calculation
+def derive_values_for_calc(path_input_emis_calc, category, fuel, segment, fuel_st, technology, pollutant):
+    # maybe we should check that all the inputs are strings before make df
+    values_for_emis_calc = pd.read_csv(path_input_emis_calc)
+    df_values = values_for_emis_calc[(values_for_emis_calc['Category'] == category) &
+                                     (values_for_emis_calc['Fuel'] == fuel) &
+                                     (values_for_emis_calc['Segment'] == segment) &
+                                     (values_for_emis_calc['Euro Standard'] == fuel_st) &
+                                     (values_for_emis_calc['Technology'] == technology) &
+                                     (values_for_emis_calc['Pollutant'] == pollutant)]
+    a = df_values['Alpha']
+    b = df_values['Beta']
+    g = df_values['Gamma']
+    d = df_values['Delta']
+    e = df_values['Epsilon']
+    z = df_values['Zita']
+    h = df_values['Hta']
+    return a, b, g, d, e, z, h
```

Comment:
You can do this in one line as follows: 
`return df_values['Alpha'], df_values['Beta'], ... etc.`

Reply:
True, I'll change it then